src/main/cpp/src/ftos_converter.cu

```diff
@@ -0,0 +1,1161 @@
+/* Not a contribution
+ * Changes made by NVIDIA CORPORATION & AFFILIATES enabling ryu or otherwise documented as
+ * NVIDIA-proprietary are not a contribution and subject to the following terms and conditions:
+ *
+ * SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ * SPDX-License-Identifier: LicenseRef-NvidiaProprietary
+ *
+ * NVIDIA CORPORATION, its affiliates and licensors retain all intellectual
+ * property and proprietary rights in and to this material, related
+ * documentation and any modifications thereto. Any use, reproduction,
+ * disclosure or distribution of this material and related documentation
+ * without an express license agreement from NVIDIA CORPORATION or
+ * its affiliates is strictly prohibited.
```

Comment:
This seems to conflict with the ASLv2 license of this project.

Reply:
It is required by the legal team. My understanding is that the `LicenseRef-NvidiaProprietary` only affects `Changes made by NVIDIA`, which is marked as `Not a contribution`, so it won't conflict with the license.
```
      "Contribution" shall mean any work of authorship, including
      the original version of the Work and any modifications or additions
      to that Work or Derivative Works thereof, that is intentionally
      submitted to Licensor for inclusion in the Work by the copyright owner
      or by an individual or Legal Entity authorized to submit on behalf of
      the copyright owner. For the purposes of this definition, "submitted"
      means any form of electronic, verbal, or written communication sent
      to the Licensor or its representatives, including but not limited to
      communication on electronic mailing lists, source code control systems,
      and issue tracking systems that are managed by, or on behalf of, the
      Licensor for the purpose of discussing and improving the Work, but
      excluding communication that is conspicuously marked or otherwise
      designated in writing by the copyright owner as "Not a Contribution."
```

Reply:
I updated the copyright and license to add our apache license. Thanks!